rules/test-title.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+/* eslint quote-props: [2, "as-needed"] */
+module.exports = function (context) {
+	var ifMultiple = context.options[0] === 'if-multiple';
```

Comment:
Should we maybe use a options schema to prevent users from doing mistakes in the config? http://eslint.org/docs/developer-guide/working-with-rules#options-schemas